pulp_smash/tests/pulp3/utils.py

```diff
@@ -218,3 +218,14 @@ def read_repo_content(repo):
     return (api
             .Client(config.get_config(), api.json_handler)
             .get(urljoin(get_latest_repo_version(repo), 'content/')))
+
+
+def get_name_content_units(repo):
```

Comment:
I think a function name like `get_content_unit_names` would be more appropriate.